rfcs/0003-kubernetes-oci/README.md

```diff
@@ -134,6 +134,19 @@ spec:
       name: cosign-key
 ```
 
+### Layer selection
+
+By default, Flux assumes that the first layer of the OCI artifact contains the Kubernetes configuration.
+For multi-layer artifacts created by other tools than Flux CLI, users can specify the media type of the layer
```

Comment:
We should add links to such tools as an example here to clarify the context of this feature.

Reply:
I think it might be better (or besides this) to also refer to the OCI spec around media types.

Reply:
xref: https://github.com/opencontainers/image-spec/blob/v1.0.2/media-types.md

In addition, we might want to be more explicit about the media type expected to be a "compressed layer" (xref: https://github.com/opencontainers/image-spec/blob/v1.0.2/layer.md#gzip-media-types). As this seems to be implied by https://github.com/google/go-containerregistry/blob/main/pkg/v1/stream/layer.go#L131-L136 which we use in the controller code.

Reply:
I think I’ve addressed the comments here, please check the last push.

Reply:
Looks good to me, thanks Stefan.